avocado/core/remote/runner.py

```diff
@@ -86,7 +89,13 @@ def check_remote_avocado(self):
         if result.exit_status == 127:
             return (False, None)
 
+        # Get the remote and local versions of avocado, and compare the major versions only
         match = self.remote_version_re.findall(result.stdout)
+        self.remote_version = '.'.join(list(match[0]))
+        self.local_version = version.VERSION
+        if self.remote_version.split('.')[0] != self.local_version.split('.')[0]:
```

Comment:
Hi @apahim  My logic to do the join is to make sure it's in the same format as what  `version.VERSION` returns. Since `match` is returned as `(36, 2)`, I do a join so it's in a string format, and split both strings to only compare the major versions